public/js/visualize.js

```diff
@@ -1,75 +1,105 @@
-var urlParams = new URLSearchParams(window.location.search);
-var url = "visualize/" + urlParams.getAll("articleID");
+const urlParams = new URLSearchParams(window.location.search);
+const url = `visualize/${urlParams.getAll('articleID')}`;
 
-fetchGET(url, function(err, data) {
+fetchGET(url, (err, data) => {
```

Comment:
This is front end code but I don't think you're transpiling back to ES5 in this project. Have you tested this in browsers that aren't so modern to have ES6 built in?

Reply:
We wrote it ES5 for frontend codes but prettier changed it to ES6 automatically which I didn't realize when I made this PR. Now all the public files are prettier-ignored!